README.md

```diff
@@ -62,15 +62,23 @@ This library supports sending alert as email and as message card to Ms Teams' ch
 
 ## Usage
 
-```javascript
-// All required configurations must be loaded to environment.
+### Imports
 
-const Notify = require('node-alertnotification');
+```js
+// javascript
+const Notifier = require('node-alertnotification');
+
+// typescript
+import Notifier from 'node-alertnotification';
+```
+### Send notification
+```js
+// All required configurations must be loaded to environment.
 
  //Create New Notify
- let notify = new Notify(new EvalError('Test notify 001'), [new EvalError('Test notify 001'), new EvalError('Test notify 002')])
+const notifier = new Notifier(new EvalError('Test notify 001'), [new EvalError('Test notify 001'), new EvalError('Test notify 002')])
```

Comment:
This needs to be a major version not minor otherwise it'd break any projects using it.
Should revert this vup and upgrade to `1.1`